services/workflows-service/src/main.ts

```diff
@@ -1,20 +1,20 @@
-import passport from 'passport';
-import cookieSession from 'cookie-session';
```

Comment:
please revert changes in this file, will be changed in my another PR
https://github.com/ballerine-io/ballerine/pull/1568/files#diff-d53b043ddda2125d7ae915af1d7b718a8800e1d48d28034fd347b093c67bf5ea
![Uploading image.png…]()